DeadCrumbs/app/src/main/java/dead/crumbs/ui/MainActivity.kt

```diff
@@ -64,6 +65,8 @@ class MainActivity : AppCompatActivity() {
 
 
     //------------Bluetooth Part-------------//
+    private val rssiProximity: RSSIProximity = RSSIProximity();
```

Comment:
Maybe have all variable at the top of the class definition or above the place it is used. Forexample this is used on line 143, but defined on line 68.